src/components/grid-aware/VideoHeader/VideoHeader.module.css

```
@@ -0,0 +1,145 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background-image: linear-gradient(90deg, #000 0%, rgba(0, 0, 0, 0) 100%), var(--background-image);
+  background-size: cover;
```

Actually, something I just thought of is that you may want to center the image.

You may want to replace this with:

```suggestion
  background:
    linear-gradient(90deg, #000 0%, rgba(0, 0, 0, 0) 100%),
    center / cover var(--background-image);
```